plugins/MantisCoreFormatting/MantisCoreFormatting.php

```diff
@@ -52,7 +67,88 @@ function config() {
 			'process_text'		=> ON,
 			'process_urls'		=> ON,
 			'process_buglinks'	=> ON,
+			'process_markdown'	=> OFF
+		);
+	}
+
+	/**
+	 * Add img-src directives on markdown to enable to referenced images from internet.
+	 * @return void
+	 */
+	function csp_headers() {
+		static $s_markdown;
+		
+		if( null === $s_markdown ) {
+			$s_markdown = plugin_config_get( 'process_markdown' );
+		}
+
+		if (ON == $s_markdown ) {
+			http_csp_add( 'img-src', "*" );
```

Comment:
I'm not a security expert, and I'm not sure what kind of implications this may have, but on principle I am also not overly keen on allowing images from anywhere.